Dockerfile

```diff
@@ -1,66 +1,26 @@
-###################
-# BUILD FOR LOCAL DEVELOPMENT
-###################
-
-FROM node:19-alpine As development
-
-# Create app directory
-WORKDIR /usr/src/app
-
-# Copy application dependency manifests to the container image.
-# A wildcard is used to ensure copying both package.json AND package-lock.json (when available).
-# Copying this first prevents re-running npm install on every code change.
-COPY --chown=node:node package*.json ./
-
-# Install app dependencies using the `npm ci` command instead of `npm install`
-RUN npm ci --legacy-peer-deps
-
-# Bundle app source
-COPY --chown=node:node . .
-
-# Use the node user from the image (instead of the root user)
-USER node
-
-###################
-# BUILD FOR PRODUCTION
-###################
-
-FROM node:19-alpine As build
-
-WORKDIR /usr/src/app
-
-COPY --chown=node:node package*.json ./
-
-# In order to run `npm run build` we need access to the Nest CLI.
-# The Nest CLI is a dev dependency,
-# In the previous development stage we ran `npm ci` which installed all dependencies.
-# So we can copy over the node_modules directory from the development image into this build image.
-COPY --chown=node:node --from=development /usr/src/app/node_modules ./node_modules
-
-COPY --chown=node:node . .
-
-# Run the build command which creates the production bundle
+# Node Alpine Docker Image
+FROM node:18.9-alpine3.16
+
+# Variables
+ARG GITHUB_SHA
+ENV GITHUB_SHA=$GITHUB_SHA
+
+# Install packages
+RUN apk add bash openrc curl \
+  && rm -rf /var/cache/apk/*
+
+# Node setup
+WORKDIR /app
+COPY package.json .
+RUN npm i --omit=dev --legacy-peer-deps
```

Comment:
Do we not want to `npm ci` from `package-lock.json` instead? With `npm i` from `package.json` without `package-lock.json`, we're getting a completely new resolution of our dependencies

Reply:
Good point, have changed it to `npm ci`.